Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/transform/subdisc/HL7PRPA201306Transforms.java

```diff
@@ -678,7 +684,7 @@ protected boolean areSenderFieldsNull(PRPAIN201305UV02 oRequest) {
 
         if (NullChecker.isNullish(oRequest.getSender().getDevice().getId().get(0).getRoot())) {
             LOG.error("The device id value (i.e. II.getRoot() or "
-                + "oRequest.getSender().getDevice().getId().get(0).getRoot()) is null.");
+                    + "oRequest.getSender().getDevice().getId().get(0).getRoot()) is null.");
```

Comment:
That update in that logging was done from the auto-formatting.  It had nothing to do with the change done for FHAC-760.